lib/govuk_publishing_components/presenters/meta_tags.rb

```diff
@@ -106,6 +107,14 @@ def add_step_by_step_tags(meta_tags)
         meta_tags
       end
 
+      def add_canonical_tag(meta_tags)
```

Comment:
I've pushed up a change. It makes the attribute configurable but if you leave it as `true` it sticks with the default base path (to avoid having to add too much logic to the frontend apps), what do you think?

Reply:
That change looks good to me 👍 - although as it's a path I think we should still prefix with Plek website root when it's configured

Reply:
Good point, I've amended that now!